src/client/core/index.js.wrapper.mustache

```diff
@@ -0,0 +1,12 @@
+(function () {
+    //NOTE: we should have an ability to initialize scripts with different contexts.
+    // It's necessary for an iframe without src attribute because the scripts are not injected
+    // inside it by hammerhead. So, we wrap all scripts with initialization functions.
```

Comment:
We should have the capability to initialize scripts with different contexts. This is required for iframes without the src attribute because Hammerhead does not inject scripts into such iframes. So, we wrap all scripts with initialization functions.